VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,97 @@
+
+Since 2010, the prevalence of the public API has skyrocketed. Smartphones finally brought APIs to our daily lives, every day, every hour, making life easier for every user that was coming and going. We now had support in understanding the world with the help of machines that communicated with each other fluently. All this information is within grasp to men and women, merely one-click away. 
+
+E-commerce, Social media, cloud computing, mobility… Without a doubt, we are definitely living in the Web API era.
+
+## Types of APIs
+
+There are 4 types of APIs: 
+
+- **Web services APIs**: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
+
+    There are four types of Web services APIs:
+
+    •	SOAP (Simple Object Access Protocol), a standard protocol for exchanging information and data in XML between two objects.
+
+    •	XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.
+
+    •	JSON-RPC, same as XML-RPC but in JSON format. 
+
+    •	REST (Representational State Transfer), a software architecture for hypermedia systems on the World Wide Web; a REST API often uses the HTTP protocol.
+
+- **Library-based APIs**: These are APIs designed for a specific programming language. They are a means to reuse libraries from other software or other parts of the same software.
+
+- **Function APIs in operating systems**: software programs are continuously interacting with operating systems (OS). In many cases, the way they do so is through APIs. Operating systems such as Windows have APIs that enable such communication between programs and the OS.
+
+## Benefits of APIs
+
+Let's highlight the many benefits of APIs:
+
+- **Adaptation**: needs change over time and APIs help anticipate changes. When working on this technology, data migration can be better supported, while offering a better review of the information. 
+
+- **Automation**: Through APIs, workflows can be updated so that they can be done in less time while being more productive.
+
+- **Flexibility**: enabling API access to app components provides flexibility in the delivery of services and information.  
+
+- **Customization**: through APIs, any user can customize content and services.
+
+- **Efficiency**: when access to an API is provided, the information is made available immediately. 
+
+- **Integration**: APIs allow information to be easily shared between applications or products. This enables an integrated user experience.
+
+## Example of an API
+
+I think it would be great to give some example of API related to AsyncAPI, any idea or suggestion?
+
```

Comment:
Nice job! I think this will turn out a great introductory video. 
If I can give add a more general commentary about the part that talks about the history of APIs is this:
Use story-telling. Structure the information around the elements of a story: exposition, conflict, climax, and conclusion. People are interested in people. Give specific names instead of talking about companies. Who were the main contributors to the origins of APIs? What were their contributions? 
Another tip is that your story should answer what? how? why? and when?